.github/workflows/test_selenium.yml

```diff
@@ -15,19 +15,19 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Conda setup
-      uses: s-weigand/setup-conda@v1.0.5
+      uses: s-weigand/setup-conda@v1
       with:
         update-conda: true
 
     - name: Conda environment creation
       shell: bash -l {0}
       run: |
-        conda create --yes --name TEST python=3.9 pip --file requirements.txt --file requirements-dev.txt --channel conda-forge --strict-channel-priority
+        conda create --yes --name TEST python="3.10" pip --file requirements.txt --file requirements-dev.txt --channel conda-forge --strict-channel-priority
```

Comment:
This may break and we may need to wait b/c probably not all pkgs are available for 3.10 yet.